tests/integration/sync/sync.spec.ts

```diff
@@ -1303,7 +1426,6 @@ describe('sync', function () {
         const response = await fileRequestSender.patchFile(syncId as string, fileId as string, { totalEntities: 0 });
 
         expect(response.status).toBe(httpStatus.OK);
-        expect(response.body).toEqual([syncId]);
```

Comment:
Hey, Omer!
Thanks for stopping by. 
Please note, this is a test for sync. That is why, we do not check the validity of `patchFile` response. It will be checked on different tests that specifically check patchFile (such as in /integration/file.spec.ts). 
Moreover, These requests are PRE-requests in order to check the validity of sync process. 
Please feel free to ask more questions on this matter. 
- Natalie and Niv😉